src/dagmc/util.hpp

```diff
@@ -0,0 +1,14 @@
+#ifndef _DAGMC_UTIL
+#define _DAGMC_UTIL
+
+#include <string>
+
+namespace moab {  // TODO: separate into a new namespace
```

Comment:
(Already mentionned it... but adding it here for simplicity) why not already introduce a `dagmc_tool` namespace ?